health/notifications/health_alarm_notify.conf

```diff
@@ -61,7 +61,7 @@
 # For RFC 1036 dates, use '+%A, %d-%b-%y %H:%M:%S %Z'
 # For a reasonably local date and time (in that order), use '+%x %X'
 # For the old default behavior (compatible with ANSI C's asctime() function), leave this empty.
-date_format=''
+date_format='+%x %X'
```

Comment:
These are default options that provide the maximum possible compatibility. No need to make these changes to the stock configuration we ship with netdata, users can define what they need. This would actually cause issues for people who need that old behavior described in the comment above.